sw/airborne/math/pprz_algebra_float.h

```diff
@@ -101,6 +101,19 @@ struct FloatRates {
     while (_a < -M_PI) _a += (2.*M_PI);             \
   }
 
+/*
+ * Returns the log of v in base of n
+ */
+static inline float float_log_n(float v, float n)
+{
+  if (v == 0.0) {
```

Comment:
Maybe also check n here.

Reply:
I'm not sure what you are referring to here

Reply:
You check v == 0 but not n == 0

Reply:
But also, float == 0 is not a good idea... See my previous comments